indexer/locals.tf

```diff
@@ -137,7 +140,10 @@ locals {
       requires_postgres_connection : true,
       requires_redis_connection : true,
       should_deploy_in_rds_subnet : false,
-      ecs_environment_variables : [],
+      ecs_environment_variables : [{
```

Comment:
`vulcan` is horizontally scalable, I don't think we should have up to 15 connections per instance as we could scale up to a point where this slows down the database.

Reply:
Maybe start with a smaller number, like 5?

Reply:
Yes... depends how many threads the database has and how many instances of `vulcan` you expect and how many parallel connections you expect a single instance to actually use

In general databases will go faster if they have fewer (tens) of connections to manage rather than hundreds. By the time you get to thousands the performance would likely degrade significantly

Reply:
removed vulcan change